### PR TITLE
Add issue template for better orientation

### PR DESCRIPTION
**Pull request by SimonFrings**
This pull request adds issue templates to this project.

grafik

With GitHub Discussions now activated on an organizational level (as discussed in #460), it makes sense to add issue templates to each ReactPHP repository to show users where they have to open up specific tickets:
- New feature ideas and questions belong in  [ReactPHP discussions](https://github.com/orgs/reactphp/discussions). GitHub discussions are more long-lived than issues, thus enabling better visibility for users.
- We'll continue to use `issues` for bug reports. These bugs will be fixed at some point and then belong in `closed issues`. Additionally I won't add a bug template to [reactphp/reactphp](https://github.com/reactphp/reactphp). IMO it makes no sense to report a bug in this repository. All bugs should be reported inside the related repo (http, socket, dns, etc.).
- Security issues should be reported to us via email. It is important not to make security-related issues public before they have been fixed!

I will file pull requests to add issue templates for each ReactPHP repository. Repos like https://github.com/reactphp/branding or https://github.com/reactphp/.github won't receive them (makes no sense).

### CODE REVIEW

**Comment by nhedger** [comment]
I believe it's possible to create default issue templates at the organization level.

- https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file

This should save you from having to open PR on all repositories.

**Comment by SimonFrings** [comment]
@nhedger You're totally right, the only thing I'd have to do is to add the `ISSUE_TEMPLATE` folder to the `.github` folder inside [reactphp/.github](https://github.com/reactphp/.github) and it's done.

The downside is that EVERY project inside the [ReactPHP organisation](https://github.com/reactphp) gets the same issue template. This may make sense for most projects (like dns, http, etc.) but not for something like [reactphp/branding](https://github.com/reactphp/branding) or [reactphp/reactphp](https://github.com/reactphp/reactphp). I would instantly do this if GitHub had some kind of functionality to exclude specific repositories, but I didn't find anything similar.

In conclusion: I am willing to take the slightly longer road and do it for every repository individually (which means only to copy-paste at some point 😄). Just feels right ^^

**Comment by SimonFrings** [comment]
@clue, @WyriHaximus I had to rebase my chnages to match #462, now all tests are green and this is ready for review.

**Review by WyriHaximus — APPROVED** [review]
:+1:

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Report a security vulnerability
+    url: https://reactphp.org/#support
+    about: 'If you discover a security vulnerability, please send us an email. Do not disclose security-related Issues publicly.'
```
Comment (nhedger) [inline .github/ISSUE_TEMPLATE/config.yml]:
```suggestion
    about: 'If you discover a security vulnerability, please send us an email. Do not disclose security-related issues publicly.'
```

Comment (SimonFrings) [reply .github/ISSUE_TEMPLATE/config.yml]: Good catch :+1:

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+blank_issues_enabled: false
```
Comment (clue) [inline .github/ISSUE_TEMPLATE/config.yml:1]: I think we should keep a section to report bugs in the templates. Although I don't expect this to be used frequently, it feels a bit weird to click on the "Add issue" button and not have the option to actually add an issue.

Comment (SimonFrings) [reply .github/ISSUE_TEMPLATE/config.yml:1]: You're right, I wasn't aware of that :D

I force-pushed and updated my screenshot above :+1:

**Review by clue — APPROVED** [review]
@SimonFrings Thank you for the update, changes LGTM! :shipit:

**Review by WyriHaximus — APPROVED** [review]
